### PR TITLE
feat(pipeline): Refactor to workflow

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Changed

- Refactor pipeline to workflow in CGDeliverables

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b refactor-pipeline-to-workflow-cgdeliverabels `

### How to test
- [ ] Do ...

### Expected test outcome
- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Review by ivadym — APPROVED** [review]
Nice, I guess we'll update the CLI later 💯

**Comment by henrikstranneheim** [comment]
> Nice, I guess we'll update the CLI later 💯

Yes, more PRs will follow.

File: cg_hermes/cli/validate.py [hunk 1]
```diff
@@ -27,11 +27,11 @@
 @app.command("deliverables")
 def validate_deliverables(
     infile: Path,
-    pipeline: Workflow = typer.Option(Workflow.FLUFFY, help="Specify pipeline"),
+    pipeline: Workflow = typer.Option(Workflow.FLUFFY, help="Specify workflow"),
```
Comment (diitaz93) [inline cg_hermes/cli/validate.py:30]: I assume the renaming of this variable is for a future PR

**Review by diitaz93 — COMMENTED** [review]
👍

File: cg_hermes/cli/validate.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg_hermes/cli/validate.py:30]: Correctly assumed. Trying to limit the scope when possible.

**Comment by henrikstranneheim** [comment]
Deployed to production:

```
Log deploy... done.
hermes version: 3.0.0
```

**Pull request merged**